test/graphql/transaction_test.exs

```diff
@@ -328,6 +330,47 @@ defmodule GodwokenExplorer.Graphql.TransactionTest do
              json_response(conn, 200)
            )
 
+    query = """
+    query {
+      transactions(
+        input: {
+           method_id: "0x12345678"
+        }
+      ) {
+        entries {
+          method_id
+          method_name
+          polyjuice {
+            input
+          }
+        }
+      }
+    }
+    """
+
+    conn =
+      post(conn, "/graphql", %{
+        "query" => query,
+        "variables" => %{}
+      })
+
+    assert match?(
```

Comment:
There should be another case that if `method id in query is not 0x12345678`, records of `method id is 0x12345678` should not be returned

Reply:
test case was added